friendica_test_data.sql

```diff
@@ -112,13 +112,13 @@ DROP TABLE IF EXISTS `cache`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `cache` (
```

Comment:
have you tested the VM running with that new DB?

Reply:
No, I don't have Vagrant on Windows and I'm not sure how to get it.